jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/corereaders/zos/util/CompressedRecordArray.java

```diff
@@ -494,7 +494,7 @@ private static void fpostest(String filename) throws Exception {
                 }
                 if (eof) break;
             }
-            long now = System.currentTimeMillis();
+            final long now = System.nanoTime();
             System.out.println("block size " + (1 << blockSizeLog2) + " uses " + ra.memoryUsage() + " took " + (now - then) + " ms");
```

Comment:
The message indicates "ms", but now it's "ns".

Reply:
Updated.